<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>820</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QVideoWidget" name="Video" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGraphicsView" name="rollView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="valueRMS">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пожалуйста, проведите калибровку!!!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="prevVideo">
         <property name="minimumSize">
          <size>
           <width>209</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="numberVideo">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 12px;
}</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="nextVideo">
         <property name="minimumSize">
          <size>
           <width>209</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="pitchView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="calibBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>Calib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="markerBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(0, 102, 204);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 0, 180);
    font: bold 14px;
}
QPushButton::pressed{
   background-color: rgb(0, 102, 255);
	border-style: inset;
	border-width: 5px;
	border-color: rgb(26, 95, 180);
}</string>
         </property>
         <property name="text">
          <string>Marker</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="pos3D">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
